### DISCUSSION TITLE
Dashboard list: Ability to provide folderUid instead of folderId

### DISCUSSION THREAD

**Discussion by cedric-lefebvre-winamp** [grafana/grafana#55736] [Plugins]
Hi,

I tried to apply the json template of a dashboard i'm building, but the `dashboard list` plugin doesn't seem to allow the folderUid parameter, and the id is auto-generated by environment. Which prevent me from using that template on multiple environment.

Is there a plan to allow this field in the near future?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
